### PR TITLE
chore(spanner): fix EOL in spanner documentation

### PR DESCRIPTION
**Pull request by sakthivelmanii**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-spanner/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes b/377182478

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: generation_config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ libraries:
     name_pretty: Cloud Spanner
     product_documentation: https://cloud.google.com/spanner/docs/
     client_documentation: https://cloud.google.com/java/docs/reference/google-cloud-spanner/latest/history
-    api_description: is a fully managed, mission-critical,  relational database service that offers transactional consistency at global scale, \nschemas, SQL (ANSI 2011 with extensions), and automatic, synchronous replication \nfor high availability.\n\nBe sure to activate the Cloud Spanner API on the Developer's Console to\nuse Cloud Spanner from your project.
+    api_description: is a fully managed, mission-critical,  relational database service that offers transactional consistency at global scale, schemas, SQL (ANSI 2011 with extensions), and automatic, synchronous replication for high availability. Be sure to activate the Cloud Spanner API on the Developer's Console to use Cloud Spanner from your project.
```
Comment (olavloite) [inline generation_config.yaml:9]: Isn't this file auto-generated by the code generator? Which means that these changes will be overwritten during the next code generation?

Comment (sakthivelmanii) [reply generation_config.yaml:9]: No. autogeneration only updates the following fields. Not the actual file. 

* googleapis_commitish
* gapic_generator_version

https://github.com/googleapis/java-spanner/blob/main/.github/scripts/update_generation_config.sh#L95-L103

Comment (olavloite) [reply generation_config.yaml:9]: ack, thanks for clarifying

**Review by olavloite — APPROVED** [review]

**Comment by blakeli0** [comment]
Please note this change didn't trigger the regeneration of READMEs since it is from a fork. Please merge the main branch to `generate-libraries-main` branch so the [nightly generation PR](https://github.com/googleapis/java-spanner/pull/3468) can pick up this change. Or you can merge the current [nightly generation PR](https://github.com/googleapis/java-spanner/pull/3468) first, the next generation PR should pick up this change as well.

**Pull request merged**